src/components/AddFood.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+
+class AddFood extends Component {
+    state = {
+        name: this.props.name,
+        calories: this.props.calories,
+        image: this.props.image,
+        foods: []
+
+    }
+
+        handleQuantityChange = (event) => {
+            console.log(event.currentTarget.value)
+        let newValue = event.currentTarget.value
+        this.setState ({
+            quantity: newValue 
+        })
+        }
+
+        handleFormSubmit = (event) => {
+            event.preventDefault() // always do this with submit events
+            let foodName = event.currentTarget.food.value
+            let image = event.currentTarget.image.value
+            let calories = event.currentTarget.calories.value
+            let newfoods = [{name:foodName, calories: calories, image:image, completed: false}, ...this.state.foods]
+            this.setState({
+              foods: newfoods,
```

Comment:
up to this point the code is perfect, but then when you get the new food you are just saving that into the state. The idea would be to use the handleAddfood you are passing through props and invoke it with the food element. That way the function will run in App.js and update the elements with the new food.